packages/expo/src/hooks/useSignInWithApple.ts

```diff
@@ -0,0 +1,69 @@
+import type { SetActive, SignInResource, SignUpResource } from '@clerk/types';
+
+import { errorThrower } from '../utils/errors';
+
+export type StartAppleAuthenticationFlowParams = {
```

Comment:
would it make sense to move these to a types file and then `export * from './useSignInWithApple.types.ts`? I'm not sure how TypeScript specifically works in instances where we're using platform-specific files, but it would eliminate having the types duplicated.